src/SwedbankPay.Sdk.Infrastructure/JsonSerialization/JsonSerialization.cs

```diff
@@ -0,0 +1,72 @@
+﻿using SwedbankPay.Sdk.JsonSerialization.Converters;
+using SwedbankPay.Sdk.PaymentInstruments;
+using SwedbankPay.Sdk.PaymentInstruments.Invoice;
+using SwedbankPay.Sdk.PaymentOrders;
+using System.Text.Json;
+
+namespace SwedbankPay.Sdk.JsonSerialization
+{
+    public static class JsonSerialization
+    {
+        private static JsonSerializerOptions settings;
+
+        public static JsonSerializerOptions Settings
+        {
+            get
+            {
+                if (settings == null)
+                {
+                    CreateJsonSerializerOptions();
+                }
+
+                return settings;
+            }
+        }
+
+        private static void CreateJsonSerializerOptions()

```

Comment:
This can just be changed to a static constructor, no?

```suggestion
        public static JsonSerialization()

```